app/models/linked_site.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: linked_sites
+#
+#  id         :integer          not null, primary key
+#  link_type  :integer
+#  name       :string
+#  share_from :boolean
+#  share_to   :boolean
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class LinkedSite < ActiveRecord::Base
+  LINK_TYPE = %w[username OAuth2].freeze
```

Comment:
`oauth2` — symbols like this shouldn't be uppercase

Also should be a `%i` literal for symbols, not `%w`

And it's not necessary to declare it as a constant, you can also just do `enum link_type: %i[username oauth2]`